test/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
 Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
please add a compat bound for Documenter. Preferably `0.27` for now, and we can adapt that when we bump it for the docs

Reply:
I don't think 1.0 behaves differently here.

Reply:
done